source/upgrade.txt

```diff
@@ -170,10 +179,11 @@ The v4.0 driver introduces the following breaking changes to the build system:
 - Drops support for the ``MONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX`` configuration
   option. Default CMake behavior for ``CMAKE_INSTALL_PREFIX`` is now respected.
 
-- Sets the ``ENABLE_TESTS`` configuration option to ``OFF`` by default. To
-  re-enable building test targets, you must set ``ENABLE_TESTS`` to ``ON``.
-  To include test targets in the "all" target, you must also set the
-  ``BUILD_TESTING`` option to ``ON``.
+.. note:: 
+
+   The :ref:`ENABLE_TESTS configuration option change
+   <version-4.1-enable-tests-config>` was originally announced for v4.0, but
+   instead was released with v4.1.
```

Comment:
This is incorrect. The 4.0 entry should be restored as it was prior, although perhaps with a note that it does not apply to auto-downloaded C Driver configurations as intended due to a bug.